qiita_pet/handlers/study_handlers/tests/test_description_handlers.py

```diff
@@ -0,0 +1,18 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from unittest import main
+
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+
+class PreprocessingSummaryHandlerTests(TestHandlerBase):
+    # TODO: Missing tests
```

Comment:
No, I just noted here because when I was splitting the file I realized that there are no tests for this function. Also, I think that there are some of these classes (like this one) that will disappear so I didn't want to spend time writing tests that are going to disappear in the next day.